cookbooks/aws-parallelcluster-environment/files/amazon-2023/network_interfaces/configure_nw_interface.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+set -ex
+
+if
+  [ -z "${DEVICE_NAME}" ] ||          # name of the device
+  [ -z "${DEVICE_NUMBER}" ] ||       # number of the device
+  [ -z "${DEVICE_IP_ADDRESS}" ] ||       # ip of the device
+  [ -z "${MAC}" ] ||                 # mac address of the device
+  [ -z "${CIDR_BLOCK}" ]                 # CIDR block of the subnet
+then
+  echo 'One or more environment variables missing'
```

Comment:
The error message should be more specific on the missed envars

Reply:
Because the same approach is being done for other OSes. https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/cookbooks/aws-parallelcluster-environment/files/default/network_interfaces/configure_nw_interface.sh